### PR TITLE
[flutter_tools] run UWP builds ahead of cmake

### PR DESCRIPTION
**Pull request by jonahwilliams**
Temporary work around for CMake task ordering. run the uwp build ahead of calling cmake. Update install manifest to have `/` paths

### CODE REVIEW

File: packages/flutter_tools/lib/src/windows/install_manifest.dart [hunk 1]
```diff
@@ -51,5 +51,5 @@ Future<void> createManifest({
   outputs.add(project.ephemeralDirectory.childFile('icudtl.dat'));
   project.ephemeralDirectory.childFile('install_manifest')
     ..createSync(recursive: true)
-    ..writeAsStringSync(outputs.map((File file) => file.absolute.path).join('\n'));
+    ..writeAsStringSync(outputs.map((File file) => file.absolute.uri.toString().split('file:///')[1]).join('\n'));
```
Comment (cbracken) [inline packages/flutter_tools/lib/src/windows/install_manifest.dart:54]: I think you could just do `uri.path` here.

**Review by cbracken — APPROVED** [review]
lgtm with one nit

**Comment by clarkezone** [comment]
LGTM

File: packages/flutter_tools/lib/src/windows/install_manifest.dart (hunk 1 above)
Comment (jonahwilliams) [reply packages/flutter_tools/lib/src/windows/install_manifest.dart]: done -ish 👍

**Comment by jonahwilliams** [comment]
g3 testing is unrelated failure that was rolled back.

**Pull request merged**